### PR TITLE
Separate input and select styles using Aphrodite

### PR DESCRIPTION
**Pull request by bsclifton**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

This is an Aphrodite friendly redo of https://github.com/brave/browser-laptop/pull/6084

Fixes https://github.com/brave/browser-laptop/issues/5972

I tested each area manually and did my best to compare to the original PR.  @luixxiul, please let me know what you think.  I did omit a two updates to select (because they looked bad):
- select box font being 1em
- select box having margin 0 5px

## screenshots of textbox / dropdown
screen shot 2017-01-16 at 3 38 14 pm

screen shot 2017-01-16 at 3 39 27 pm

## Updates to about:styles
screen shot 2017-01-16 at 3 41 08 pm

screen shot 2017-01-16 at 3 41 18 pm

### CODE REVIEW

File: app/renderer/components/textbox.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+class RecoveryKeyTextbox extends ImmutableComponent {
+  render () {
+    return <SettingTextbox data-isRecoveryKey='true' {...this.props} />
+  }
+}
+
+const styles = StyleSheet.create({
+  'formControl': {
```
Comment (luixxiul) [inline app/renderer/components/textbox.js]: Can we share this with [that](https://github.com/brave/browser-laptop/pull/6689/files#diff-fc51489835db20322f43545a85d88849R40) of app/renderer/components/dropdown.js? I'm wondering if setting them in each file would make a possibility of style inconsistency.

**Comment by luixxiul** [comment]
https://travis-ci.org/brave/browser-laptop/builds/192513240#L4619

> SyntaxError: /home/travis/build/brave/browser-laptop/app/extensions/brave/img/caret_down_grey.svg: Unexpected token, expected } (1:90)

File: app/renderer/components/textbox.js (hunk 1 above)
Comment (bsclifton) [inline app/renderer/components/textbox.js]: @luixxiul we sure can; implemented this feedback with https://github.com/brave/browser-laptop/pull/6689/commits/46d07ec1c53fa78be15906e1d24ffc391d4bd58e

File: app/renderer/components/styles/commonStyles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const {StyleSheet} = require('aphrodite')
+const globalStyles = require('./global')
+
+const styles = StyleSheet.create({
```
Comment (luixxiul) [inline app/renderer/components/styles/commonStyles.js]: How about adding these under this line as a note? Is it too much?
````
/ *
app/renderer/components/dropdown.js
app/renderer/components/textbox.js
*/
````

**Comment by luixxiul** [comment]
Also we should remove properties which are no longer required in LESS files thanks to the change. I'll mention them.

File: app/renderer/components/styles/commonStyles.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  'formControl': {
```
Comment (luixxiul) [inline app/renderer/components/styles/commonStyles.js]: https://github.com/brave/browser-laptop/blob/3d1db35c0aedb203f40b3936933a9de487e59835/less/forms.less#L8

File: app/renderer/components/textbox.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  'outlineable': {
+    ':focus': {
+      outlineColor: globalStyles.color.statsBlue,
+      outlineOffset: '-4px',
+      outlineStyle: 'solid',
+      outlineWidth: '1px'
+    }
+  },
+  'isSettings': {
+    width: '280px'
```
Comment (luixxiul) [inline app/renderer/components/textbox.js]: https://github.com/brave/browser-laptop/blob/baff549a8e56b86f8bc0a45f80869a2c35281c60/less/about/preferences.less#L373

File: app/renderer/components/textbox.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  },
+  'readOnly': {
+    background: globalStyles.color.veryLightGray,
+    boxShadow: 'none',
+    outline: 'none'
+  },
+  'recoveryKeys': {
```
Comment (luixxiul) [inline app/renderer/components/textbox.js]: https://github.com/brave/browser-laptop/blob/baff549a8e56b86f8bc0a45f80869a2c35281c60/less/about/preferences.less#L724-L727

File: js/about/preferences.js [hunk 6]
```diff
@@ -1248,16 +1219,17 @@ class PaymentsTab extends ImmutableComponent {
               <td>
                 <SettingsList>
                   <SettingItem>
-                    <select className='form-control fundsSelectBox'
+                    <FormDropdown
```
Comment (luixxiul) [inline js/about/preferences.js]: Was this applied here in some way? https://github.com/brave/browser-laptop/blob/baff549a8e56b86f8bc0a45f80869a2c35281c60/less/about/preferences.less#L1101

**Comment by luixxiul** [comment]
Feedbacks left.

File: js/about/preferences.js (hunk 6 above)
Comment (bsclifton) [inline js/about/preferences.js]: no; I manually tested and it doesn't do anything (it still stays ~10px). Also, the "Add funds" button has a similar padding. If you remove padding from "Add funds" button (ex: set to 0), it looks bad

**Comment by bsclifton** [comment]
@luixxiul I removed styles per your feedback; also I updated the Shields panel to use this new style:
screen shot 2017-01-23 at 10 14 41 am

**Comment by luixxiul** [comment]
I think the SVG merged with this PR is causing an error on Travis:

````
9) Preferences component "before all" hook:
     SyntaxError: /home/travis/build/brave/browser-laptop/app/extensions/brave/img/caret_down_grey.svg: Unexpected token, expected } (1:90)
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32.86 17.81"><defs><style>.cls-1{fill:gray;}</style></defs><title>caret_down_grey</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1_copy" data-name="Layer 1 copy"><path class="cls-1" d="M15.3,17.35.49,2.77A1.62,1.62,0,0,1,1.62,0H31.24a1.62,1.62,0,0,1,1.14,2.77L17.57,17.35A1.62,1.62,0,0,1,15.3,17.35Z"/></g></g></svg>
````

**Comment by bsclifton** [comment]
@luixxiul you are correct; I'm going to need some help w/ it. Let me ask around on Slack

**Pull request merged**